yb-voyager/cmd/exportDataStatus_test.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/testutils"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils/sqlname"
+)
+
+func TestExportSnapshotStatusStructs(t *testing.T) {
+	// Define the expected structure for TableExportStatus
+	expectedTableExportStatus := struct {
+		TableName                string `json:"table_name"`
+		FileName                 string `json:"file_name"`
+		Status                   string `json:"status"`
+		ExportedRowCountSnapshot int64  `json:"exported_row_count_snapshot"`
+	}{}
+
+	// Define the expected structure for ExportSnapshotStatus
+	expectedExportSnapshotStatus := struct {
+		Tables map[string]*TableExportStatus `json:"tables"`
+	}{}
+
+	t.Run("Validate TableExportStatus Struct Definition", func(t *testing.T) {
+		testutils.CompareStructs(t, reflect.TypeOf(TableExportStatus{}), reflect.TypeOf(expectedTableExportStatus), "TableExportStatus")
+	})
+
+	t.Run("Validate ExportSnapshotStatus Struct Definition", func(t *testing.T) {
+		testutils.CompareStructs(t, reflect.TypeOf(ExportSnapshotStatus{}), reflect.TypeOf(expectedExportSnapshotStatus), "ExportSnapshotStatus")
+	})
+}
+
+func TestExportSnapshotStatusJson(t *testing.T) {
+	// Create a table list of type []sqlname.NameTuple
+	o1 := sqlname.NewObjectName(POSTGRESQL, "public", "public", "table1")
+	o2 := sqlname.NewObjectName(POSTGRESQL, "public", "schema1", "table2")
+	nameTuple1 := sqlname.NameTuple{CurrentName: o1, SourceName: o1, TargetName: o1}
+	nameTuple2 := sqlname.NameTuple{CurrentName: o2, SourceName: o2, TargetName: o2}
+	tableList := []sqlname.NameTuple{
+		nameTuple1,
+		nameTuple2,
+	}
+
+	exportDir = os.TempDir() + "/export_snapshot_status_test"
```

Comment:
filepath.Join()

Reply:
How about defer `os.RemoveAll(exportDir)` instead of taking care of it in child functions